dataColl/schema.sql

```diff
@@ -1,9 +1,106 @@
 -- Drop existing objects
 DROP TABLE IF EXISTS Organisation;
+DROP TABLE IF EXISTS form_data;
+DROP TABLE IF EXISTS enquires;
+DROP TABLE IF EXISTS support_provided;
+DROP TABLE IF EXISTS cases_not_police_report;
+DROP TABLE IF EXISTS support_age;
+DROP TABLE IF EXISTS ethnicity;
+DROP TABLE IF EXISTS gender;
+DROP TABLE IF EXISTS cases_related_to;
 
 -- Create tables
 CREATE TABLE Organisation (
     -- A DDPO
     Id INTEGER PRIMARY KEY,
     Name TEXT NOT NULL
 );
+
```

Comment:
As you said, I think we can look at the schema separately. It shouldn't hold up the front end.

Reply:
There is definitely improvements that can be done with the tables, I think maybe the best way is to merge this in as mvp and then someone with more knowledge clean them up!